### PR TITLE
[AUD-972] Add $AUDIO screen skeleton

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add the skeleton for $AUDIO page and wire into nav.
TODO: The tiles themselves

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Locally vs. staging
Screen Shot 2022-02-28 at 8 46 20 AM
Screen Shot 2022-02-28 at 8 46 14 AM

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

Mobile staging app

### CODE REVIEW

**Review by dylanjeffers — APPROVED** [review]
This looks great! It may make sense to use the `Screen` component for audio-rewards page?

File: packages/mobile/src/screens/audio-screen/AudioScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: screenHeight
+  },
+  tiles: {},
+  tileRoot: {
+    margin: 12
```
Comment (Kyle-Shanks) [inline packages/mobile/src/screens/audio-screen/AudioScreen.tsx:23]: can use spacing here

**Review by Kyle-Shanks — COMMENTED** [review]
looking good 👍

**Comment by dylanjeffers** [comment]
> > This looks great! It may make sense to use the `Screen` component for audio-rewards page?
> 
> I followed suit with the other top level things (like favorites) b/c this has a `<Header>` and doesn't hijack the nav bar. I didn't see others that were using `<Screen>` that did the same thing. Should we update all of those?

Yeah good catch! those didnt have it cause it hadn't been built yet, thinking we probably go back and update those too. can add a linear for it!

File: packages/mobile/src/screens/audio-screen/AudioScreen.tsx (hunk 1 above)
Comment (raymondjacobson) [reply packages/mobile/src/screens/audio-screen/AudioScreen.tsx]: it is bothersome to me that borderRadius & spacing are not related now. we must fix!!!

**Pull request merged**